CMakeLists.txt

```diff
@@ -26,8 +26,8 @@ set (CPACK_PACKAGE_VERSION_MINOR ${GLOG_MINOR_VERSION})
 set (CPACK_PACKAGE_VERSION_PATCH ${GLOG_PATCH_VERSION})
 set (CPACK_PACKAGE_VERSION ${GLOG_VERSION})
 
-option (WITH_GFLAGS "Use gflags" ON)
-option (WITH_THREADS "Enable multithreading support" ON)
+option (WITH_GFLAGS "Use gflags" OFF)
+option (WITH_THREADS "Enable multithreading support" OFF)
```

Comment:
You can simply ignore the warnings about missing `gflags-config.cmake`. There are also CMake means for disabling packages using the command-line. You'll have to check the CMake documentation.

Are there any other warnings/errors that prevent generation? A log output would be helpful.